<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XML Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="backButton">
      <property name="text">
       <string>&lt;- Back</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="openButton">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="verifyButton">
        <property name="text">
         <string>Verify</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fixButton">
        <property name="text">
         <string>Fix Errors</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="formatButton">
        <property name="text">
         <string>Format</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="jsonButton">
        <property name="text">
         <string>To JSON</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="minifyButton">
        <property name="text">
         <string>Minify</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="compressButton">
        <property name="text">
         <string>Compress</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="decompressButton">
        <property name="text">
         <string>Decompress</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QTextEdit" name="xmlInput">
       <property name="placeholderText">
        <string>Enter or load XML content here...</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="xmlOutput">
       <property name="placeholderText">
        <string>Output or logs appear here...</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
